eras/shelley/test-suite/test/Test/Cardano/Ledger/Shelley/Serialisation/Golden/Encoding.hs

```diff
@@ -1309,11 +1309,11 @@ tests =
           expectedHex =
             "8700a1581ce0a714319812c3f773ba04ec5d6b3ffcd5aad85006805b047b0825"
               <> "410aa1581ca646474b8f5431261506b6c273d307c7569a4eb6c96b42dd4a2952"
-              <> "0a0386821927101903e8838284a0a0a0a08482a0a0a0a084a0a0000085a18258"
+              <> "0a0386821927101903e8828284a0a0a0a08482a0a0a0a084a0a0000085a18258"
               <> "20ee155ace9c40292074cb6aff8c9ccdd273c81648ff1149ef36bcea6ebb8a3e"
               <> "250082583900cb9358529df4729c3246a2a033cb9821abbfd16de4888005904a"
               <> "bc410d6a577e9441ad8ed9663931906e4d43ece8f82c712b1d0235affb060a19"
-              <> "03e80182a0a082a0a0f68483a0a0a083a0a0a083a0a0a0009200000019080000"
+              <> "03e80182a0a082a0a08483a0a0a083a0a0a083a0a0a0009200000019080000"
```

Comment:
I guess this is probably just un-donig whatever we had to do to add `()` to the PPUP state in conway?

Reply:
Correct